### DISCUSSION TITLE
I have a project that does not have these two plugins installed: ‘@babel/plugin-proposal-optional-chaining’, ‘@babel/plugin-proposal-nullish-coalescing-operator’; why can I use ??. and xx.? syntax directly, is it because quasar did some configuration?

### DISCUSSION THREAD

**Discussion by suzhenghui-sky** [quasarframework/quasar#16307] [CLI - PWA mode] [answered]
I have a project that does not have these two plugins installed: ‘@babel/plugin-proposal-optional-chaining’, ‘@babel/plugin-proposal-nullish-coalescing-operator’; why can I use ??. and xx.? syntax directly, is it because quasar did some configuration?

**Comment by rstoenescu** [comment, marked as answer]
For q/app-webpack, you can check out our `@quasar/babel-preset-app` here where you can see exactly what we are doing by default: https://github.com/quasarframework/quasar/blob/dev/utils/babel-preset-app/index.js

**Reply by suzhenghui-sky** [reply]
Why can't I do this demo；https://github.com/suzhenghui-sky/quasar-demo.git       In the previous project, node_modules and yarn.lock were removed, and then yarn install was not possible.

**Reply by rstoenescu** [reply]
Add your required babel plugins to babel.config.js and install them.

**Reply by suzhenghui-sky** [reply]
You don't even have to download it, you just click on my project and it comes with @quasar/babel-preset-app; By download you mean npm install/yarn install?
